pubspec.yaml

```diff
@@ -36,7 +36,7 @@ dependencies:
   flutter_svg: ^2.0.10+1
   intl: ^0.19.0
   share_plus: ^10.0.2
-  sentry_flutter: ^8.14.2
+  sentry_flutter: ^9.8.0
```

Comment:
this breaks the current iOS CI build, but makes Android shut up about deprecations :disappointed: